i18n/en.json

```diff
@@ -7311,6 +7315,14 @@
     "id": "ent.compliance.licence_disable.app_error",
     "translation": "Compliance functionality disabled by current license. Please contact your system administrator about upgrading your enterprise license."
   },
+  {
+    "id": "ent.compliance.run_export.product_boards.appErr",
```

Comment:
Hey Miguel, quick note that we don't translate messages that don't appear in the UI. If these translations are going to appear only in the console, then we can use `model.NoTranslation` key to avoid adding the translations.

We are trying to avoid adding more translation load for the translators.